app/src/main/java/org/lfenergy/compas/sitipe/rest/v1/model/BayTypicalResponse.java

```diff
@@ -82,37 +87,26 @@ public static class BayTypical {
         private String referenceAccessId;
 
         public BayTypical(
-            final int id,
-            final String accessId,
-            final String name,
-            final String version,
-            final String description,
-            final int released,
-            final String lockedBy,
-            final Long lockedOn,
-            final Long modifiedOn,
-            final String smrFile,
-            final String contentVersion,
-            final String referenceAccessId
+                final org.lfenergy.compas.sitipe.data.entity.BayTypical bt
```

Comment:
this is done, because both classes are named BayTypical. the Entity (database) is a baytypical. I renamed the BayTypicalResponse.BayTypical class to BayTypicalItem